pypesto/optimize/optimizer.py

```diff
@@ -31,6 +31,12 @@
 except ImportError:
     cma = None
 
+try:
+    import scipy
+except ImportError:
+    scipy = None
+
+
```

Comment:
scipy is an install requirement, so this should not be needed, right?

Reply:
Done